scripts/minikube.sh

```diff
@@ -216,6 +244,8 @@ up)
     if [[ "${VM_DRIVER}" == "none" ]]; then
         mkdir -p "$HOME"/.kube "$HOME"/.minikube
         install_kubectl
+    elif [[ "${VM_DRIBER}" == "podman" ]]; then
```

Comment:
maybe things work better without typo? `VM_DRIVER`...

Reply:
:facepalm:

Reply:
with this addressed, it seems to work :partying_face: (at least in manual testing)